t/lib/server.lua

```diff
@@ -449,7 +449,7 @@ end
 
 function _M._well_known_openid_configuration()
     local t = require("lib.test_admin")
-    local openid_data = t.read_file("t/plugin/openid-configuration.json")
+    local openid_data = t.read_file("t/plugin/openid/configuration.json")
```

Comment:
What about moving it to `plugin/openid-connect`? We don't have a plugin called openid.